### PR TITLE
[14.0] Add module mrp_lot_on_hand_first

### PR DESCRIPTION
**Pull request by grindtildeath**
Depends on:
 - [x] https://github.com/OCA/stock-logistics-workflow/pull/1097
 - [x] https://github.com/OCA/stock-logistics-workflow/pull/1096

### CODE REVIEW

**Comment by simahawk** [comment]
LG. Test requirements missing.

**Review by sebalix — APPROVED** [review]

File: mrp_lot_on_hand_first/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{
+    "name": "Mrp Lot On Hand First",
+    "summary": "Allows to display lots on hand first in M2o fields",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Manufacturing/Manufacturing",
+    "website": "https://github.com/OCA/manufacture",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "maintainers": ["grindtildeath"],
+    "license": "AGPL-3",
```
Comment (sebalix) [inline mrp_lot_on_hand_first/__manifest__.py:12]: Switch to LGPL? (and the dependencies)

**Review by ivantodorovich — APPROVED** [review]

File: mrp_lot_on_hand_first/__manifest__.py (hunk 1 above)
Comment (grindtildeath) [reply mrp_lot_on_hand_first/__manifest__.py:12]: Not necessarily needed if we intend to publish what's depending on AGPL.

**Pull request merged**